pkg/network/ovn_kubernetes.go

```diff
@@ -242,12 +246,23 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 
 	controlPlaneReplicaCount, _ := strconv.Atoi(rcD.ControlPlane.Replicas)
 
-	err := wait.PollImmediate(5*time.Second, 280*time.Second, func() (bool, error) {
+	var heartBeat int
+
+	err := wait.PollImmediate(OVN_MASTER_DISCOVERY_POLL, OVN_MASTER_DISCOVERY_TIMEOUT, func() (bool, error) {
 		matchingLabels := &client.MatchingLabels{"node-role.kubernetes.io/master": ""}
 		if err := kubeClient.List(context.TODO(), masterNodeList, matchingLabels); err != nil {
 			return false, err
 		}
-		return len(masterNodeList.Items) != 0 && controlPlaneReplicaCount == len(masterNodeList.Items), nil
+		if len(masterNodeList.Items) != 0 && controlPlaneReplicaCount == len(masterNodeList.Items) {
+			return true, nil
+		}
+
+		heartBeat++
+		if heartBeat%3 == 0 {
+			klog.V(2).Infof("Wating to complete bootstrapping OVN: found (%d) master nodes out of (%d) expected: timing out in %d seconds",
+				len(masterNodeList.Items), controlPlaneReplicaCount, int(OVN_MASTER_DISCOVERY_TIMEOUT)-int(OVN_MASTER_DISCOVERY_POLL)*5)
```

Comment:
@JacobTanenbaum shouldn't the OVN_MASTER_DISCOVERY_POLL be `heartBeat`?

Reply:
Actually, maybe OVN_MASTER_DISCOVERY_* should just be ints (eg `const XXXX int = 5`) and then multiple by time.Second for the PollImmediate and then you don't have to do casting in the Infof alongside math.

Reply:
done